libs/googlePayments.js

```diff
@@ -71,30 +64,24 @@ api.findAffectedUsers = function findAffectedUsers (habitrpgUsers, lastId, jobSt
     };
   }
 
-  console.log('Run query', query);
-
   let usersFoundNumber;
-
+  let newLastId;
   return habitrpgUsers.find(query, {
-    sort: {_id: 1},
+    sort: { _id: 1 },
     limit: USERS_BATCH,
     fields: ['_id', 'apiToken', 'purchased.plan'],
```

Comment:
I removed the block check from Apple. Thinking about it, we don't really want that. If an account is blocked, we'd still want to (properly) cancel their subscription. Otherwise it will get checked over and over.

Moving more code into a lib to avoid code duplication sounds smart. But I would do that in a separate PR.